pkg/root.go

```diff
@@ -67,26 +68,32 @@ var RootCmd = &cobra.Command{
 		}
 
 		// Our token has either expired or doesn't exist.
-		provider, err := oidc.NewProvider(context.Background(), viper.GetString("auth.dex_host"))
+		provider, err := oidc.NewProvider(context.Background(), viper.GetString("identity_host"))
 		if err != nil {
 			log.Fatalf("error while creating new oidc provider %v", err)
 		}
 		oauth2Config := oauth2.Config{
-			ClientID:     viper.GetString("auth.client_id"),
-			ClientSecret: viper.GetString("auth.client_secret"),
-			RedirectURL:  viper.GetString("auth.callback_url"),
+			ClientID:     C.GetClientID(),
```

Comment:
I think these should come directly from the configuration? as this is configuring your local client